iron-component-page.html

```diff
@@ -195,7 +195,7 @@
       }
     </style>
 
-    <hydrolysis-analyzer id="analyzer" src="[[_srcUrl]]" transitive="[[transitive]]" clean analyzer="{{_hydroDesc}}" loading="{{_hydroLoading}}"></hydrolysis-analyzer>
+    <hydrolysis-analyzer id="analyzer" src="[[_srcUrl]]" transitive="[[transitive]]" clean analyzer="{{_hydroDesc}}" loading="{{_hydroLoading}}" resolver="permissive"></hydrolysis-analyzer>
```

Comment:
Setting resolver to "permissive" results in “No documentation found.” in the page. `this._analyzer` is empty, contains zero elements and behaviors.

Looking forward for this PR to be merged.